internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -842,7 +842,7 @@ func MountOverlayFilesystems(seq runtime.Sequence, data interface{}) (runtime.Ta
 // SetupSharedFilesystems represents the SetupSharedFilesystems task.
 func SetupSharedFilesystems(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionFunc, string) {
 	return func(ctx context.Context, logger *log.Logger, r runtime.Runtime) (err error) {
-		targets := []string{"/", "/var/lib/kubelet", "/etc/cni", "/run"}
+		targets := []string{"/", "/var", "/etc/cni", "/run"}
```

Comment:
this seems like a big change... not sure if we need that?

Reply:
This line specifically or the whole PR?

This line specifically should match the mounts we create. Otherwise this will fail since `/var/lib/kubelet` is no longer a mount.

Reply:
ok, I think I figured it out, this code actually runs _only_ for container mode, I had an assumption it runs always